DOC-2737 - Remove specific Kubernetes version references from containerd 2.0 requirements (VMMA) (#10047)
spectrocloud/librarium: docs/docs-content/vm-management/vm-migration-assistant/create-vm-migration-assistant-profile.md

<!-- prettier-ignore-end -->

- If your OS pack/image uses [containerd](https://containerd.io/) v2, you must set the
  `device_ownership_from_security_context = true` parameter and value in the

>   `/etc/containerd/conf.d/device-ownership.toml` file through the Kubernetes layer configuration in the cluster profile

Reviewer: ```suggestion
  `/etc/containerd/conf.d/device-ownership.toml` file using the Kubernetes layer of your cluster profile
```
